src/builder/Build.kt

```diff
@@ -68,24 +65,21 @@ class Build(val waves: Map<Int, Wave>) {
     }
 
     fun survivability(wave: Wave): String {
-        val secondsToKill = wave.totalHp / totalDps
-        val leftHp = totalHp - (secondsToKill * wave.totalDps)
-        var result = ""
-        if (leftHp > 0) {
+        val calc = BattleCalc(global, lane.fighters, (0 until wave.amount).map { wave.creatures.first() })

```

Comment:
use `getFighters` here. Otherwise you also will get fights which are not build yet or sold/upgraded in this level. `getFighters` takes care of this